cmd/ef_tests/levm/runner/levm_runner.rs

```diff
@@ -88,6 +88,28 @@ pub fn prepare_vm_for_tx(vector: &TestVector, test: &EFTest) -> Result<VM, EFTes
         .map(|arg| (arg.address, arg.storage_keys.clone()))
         .collect();
 
+    // Check if the tx has the authorization_lists field implemented by eip7702.
+    let authorization_list = if let Some(list) = &tx.authorization_list {
+        // Used to convert from EFTestAuthorizationListItem to AuthorizationList = Vec<AuthorizationTuple>
+        let auth_list = list
+            .iter()
+            .map(|auth_tuple| AuthorizationTuple {
+                chain_id: auth_tuple.chain_id,
+                address: auth_tuple.address,
+                nonce: auth_tuple.nonce,
+                v: auth_tuple.v,
+                r_signature: auth_tuple.r,
+                s_signature: auth_tuple.s,
+                // If the signer is not present, set it to Address::zero()
+                signer: auth_tuple.signer.unwrap_or_default(),
+            })
+            .collect::<Vec<AuthorizationTuple>>();
+
+        Some(auth_list)
+    } else {
+        None
+    };
```

Comment:
Nit. When you have an `Option<T>` and you want to perform a transformation of `T` to finally return an `Option<T'>` you can `map` the current `Option<T>`. It will be `Some<T'>` if `Option<T>::is_some() == true` or `None` if `Option<T> == None`.
```suggestion
    let authorization_list = tx.authorization_list.clone().map(|list| {
        list.iter()
            .map(|auth_tuple| AuthorizationTuple {
                chain_id: auth_tuple.chain_id,
                address: auth_tuple.address,
                nonce: auth_tuple.nonce,
                v: auth_tuple.v,
                r_signature: auth_tuple.r,
                s_signature: auth_tuple.s,
                // If the signer is not present, set it to Address::zero()
                signer: auth_tuple.signer.unwrap_or_default(),
            })
            .collect::<Vec<AuthorizationTuple>>()
    });
```